zathura/utils.c

```diff
@@ -134,9 +141,11 @@ void index_scroll_to_current_page(zathura_t* zathura) {
   gtk_tree_model_foreach(model, search_current_index, &search_data);
 
   GtkTreePath* current_path = gtk_tree_model_get_path(model, &search_data.current_iter);
+  zathura->global.current_index_path = gtk_tree_path_to_string(current_path);
```

Comment:
Does this need to be freed at some point?

Reply:
I assume here that you want for it to be freed each time a new value is assigned

Reply:
Yes, and when cleaning up.